src/main/java/com/google/enterprise/cloudsearch/fs/FsRepository.java

```diff
@@ -1117,6 +1156,17 @@ private void getDirectoryStreamContent(
           log.log(Level.WARNING, "Skipping {0} because {1}.", new Object[] {path, e.getMessage()});
           continue;
         }
+
+        try {
```

Comment:
I thought that we had said we weren't going to filter when pushing children since we now needed to know the types. My concern is that we had serious performance issues in the adaptor when looking too closely at the children in the past.